src/tokens/token.ts

```diff
@@ -238,6 +242,8 @@ export const tokenDataByNetwork: Record<
     dFRAX: "deploy me",
 
     GEAR: "0x3321F5dA65165042903eDe71617F912942f4E70F",
+
+    OHM: ADDRESS_0X0,
```

Comment:
Should have everything either as `deploy me` or address(0), so it can be handled in bindings